site/src/api/api.ts

```diff
@@ -428,15 +428,21 @@ export const getEntitlements = async (): Promise<TypesGen.Entitlements> => {
   return response.data
 }
 
-interface GetAuditLogsOptions {
-  limit: number
-  offset: number
-}
-
 export const getAuditLogs = async (
-  options: GetAuditLogsOptions,
+  options: TypesGen.AuditLogsRequest,
 ): Promise<TypesGen.AuditLogResponse> => {
-  const response = await axios.get(`/api/v2/audit?limit=${options.limit}&offset=${options.offset}`)
+  const searchParams = new URLSearchParams()
+  if (options.limit) {
+    searchParams.set("limit", options.limit.toString())
+  }
+  if (options.offset) {
+    searchParams.set("offset", options.offset.toString())
+  }
+  if (options.q) {
```

Comment:
nit: is `q` short for `query`? would be nice to spell it out

Reply:
Yeah, but it is auto-generated 😕

Reply:
But don't we have control of what to call it on the backend?